### PR TITLE
fix: default build strategy patch

### PR DESCRIPTION
**Pull request by matejvasek**

### CODE REVIEW

File: openshift/patches/0002-default-build.patch [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+@@ -7,7 +7,7 @@ import (
+ const (
+ 	BuilderPack    = "pack"
+ 	BuilderS2i     = "s2i"
+-	DefaultBuilder = BuilderPack
++	DefaultBuilder = BuilderS2i
+ )
  
--	cmd.Flags().StringP("builder", "b", "pack", "build strategy to use when creating the underlying image. Currently supported build strategies are 'pack' and 's2i'.")
-+	cmd.Flags().StringP("builder", "b", "s2i", "build strategy to use when creating the underlying image. Currently supported build strategies are 'pack' and 's2i'.")
```
Comment (zroubalik) [inline openshift/patches/0002-default-build.patch:10]: https://github.com/knative-sandbox/kn-plugin-func/blob/df022f5c93911687e4356f4c58ccbb5d10b7a895/cmd/build.go#L52

This line has been changed upstream, so it shouldn't be needed to change the default value in here, once the `DefaultBuilder` is being patched.

Comment (matejvasek) [reply openshift/patches/0002-default-build.patch:10]: Well we are not changing it, that's what this PR does, am I misunderstanding something?

Comment (matejvasek) [reply openshift/patches/0002-default-build.patch:10]: Just look at the file directly https://github.com/openshift-knative/kn-plugin-func/blob/37b25e37c3e458c9c7b39341533ac3dca5738640/openshift/patches/0002-default-build.patch

Comment (matejvasek) [reply openshift/patches/0002-default-build.patch:10]: You may be confused by the this is diff of a diff.

**Review by zroubalik — APPROVED** [review]
/lgtm

File: openshift/patches/0002-default-build.patch (hunk 1 above)
Comment (matejvasek) [reply openshift/patches/0002-default-build.patch:10]: It confuses me too many times.

**Comment by matejvasek** [comment]
This PR is related to https://github.com/knative-sandbox/kn-plugin-func/pull/1136

**Pull request merged**